core/src/edu/asu/c3simulator/util/PlayerInterfaceImplementation.java

```diff
@@ -0,0 +1,89 @@
+package edu.asu.c3simulator.util;
+
+import edu.asu.c3simulator.simulation.C3Simulation;
+import edu.asu.c3simulator.simulation.Player;
+import edu.asu.c3simulator.simulation.SocioeconomicStatus;
+
+/**
+ * Implementation of the Player interface Provides detail about various player
+ * attributes that are needed for PlayerStatusDisplay
+ * 
+ * @author Reigel, Justin
+ *
+ */
+
+public class PlayerInterfaceImplementation implements Player
+{
+	private SocioeconomicStatus status;
+	private int netWorth;
+	private int capital;
+	private C3Simulation simulation;
+
+	public PlayerInterfaceImplementation(SocioeconomicStatus newStatus, int newNetWorth,
+			int newCapital, C3Simulation newSimulation)
+	{
+		status = newStatus;
+		netWorth = newNetWorth;
+		capital = newCapital;
+		simulation = newSimulation;
+	}
```

Comment:
I just realized that this is the live data layer, not a stub...

We need to address the following issues (and possible more) and decide if the player is responsible for these or if another class will take care of them, and if so, how:
* When/how is netWorth updated? 
  * I think calculating it based on live data would be more robust and less prone to defects, but there may be a better way.
* When/how is status updated? 
* How is status determined?
* When/how is capital updated?
* How do I get a list of companies the player owns?
* Does the player have any other features, such as a line of credit or assets?
* Is it practical to allow players to exist in different simulations? 
  * I think simulation may be singleton, static, or similar
* Is the player class just a structure, or is it responsible for certain calculations or activities?